claws/openclaw/sageox/test-skill.py

```
@@ -0,0 +1,203 @@
+import time
+from datetime import datetime
+from pathlib import Path
+
+# Force unbuffered stdout so output streams in real-time
+sys.stdout.reconfigure(line_buffering=True)
+sys.stderr.reconfigure(line_buffering=True)
+
+# --- Config ---
+VPS_KEY = Path.home() / ".ssh" / "ox-bot.pem"
+VPS_HOST = "ubuntu@35.164.224.156"
+OUTPUT_DIR = Path("/tmp/sageox-test-results")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/sageox-test-results"

(S108)